pontoon/sync/tasks.py

```diff
@@ -297,7 +311,11 @@ def sync_translations(
                 changeset = ChangeSet(db_project, vcs_project, now, locale)
                 update_translations(db_project, vcs_project, locale, changeset)
                 changeset.execute()
-                update_translated_resources(db_project, vcs_project, locale)
+
+                new_entities = new_entities + changeset.new_entities
```

Comment:
`changeset.new_entities` should always be an empty list when syncing translations, so we should only collect new entities during the `sync_sources` task.